stage/dotnet/dotnet-sdk/src/Client/RequestAdapter.cs

```diff
@@ -1,17 +1,42 @@
 // Copyright (c) GitHub 2023-2024 - Licensed as MIT.
 
+using GitHub.Repos.Item.Item.Pages.Builds;
```

Comment:
I believe this reference can be removed.

Reply:
Ah yeah that was an auto-artifact of my testing.